giraffe/src/components/DapperScrollbars.tsx

```diff
@@ -106,10 +92,21 @@ export const DapperScrollbars: FunctionComponent<DapperScrollbarsProps> = ({
     setScrollLeftPos(Number(scrollLeft))
   }, [scrollTop, scrollLeft])
 
-  const dapperScrollbarsClass = classnames('cf-dapper-scrollbars', {
+  let dapperScrollbarsClasses = classnames('cf-dapper-scrollbars', {
     'cf-dapper-scrollbars--autohide': autoHide,
-    [`${className}`]: className,
   })
+    .split(' ')
+    .reduce((accum, current) => {
+      if (styles[current]) {
+        return accum ? `${accum} ${styles[current]}` : `${styles[current]}`
+      }
+      return accum ? `${accum} ${current}` : `${current}`
+    }, '')
```

Comment:
Yes it is doing something `classnames` cannot. It has to reference a style inside the `styles` object to allow CSS Modules to work correctly, applying the appropriate styles.

The styling work (and this PR in general) took a long time because of issues like this. I was seeing styles not being applied even though `classnames` had them. And then i figured out that CSS Modules needed to be used like this.